app/Nova/Team.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+namespace App\Nova;
+
+use Laravel\Nova\Panel;
+use Laravel\Nova\Fields\ID;
+use Illuminate\Http\Request;
+use Laravel\Nova\Fields\Text;
+use Laravel\Nova\Fields\Boolean;
+use Laravel\Nova\Fields\HasMany;
+use Laravel\Nova\Fields\DateTime;
+use Laravel\Nova\Fields\Textarea;
+
+class Team extends Resource
+{
+    /**
+     * The model the resource corresponds to.
+     *
+     * @var string
+     */
+    public static $model = 'App\Team';
+
+    /**
+     * The single value that should be used to represent the resource when being displayed.
+     *
+     * @var string
+     */
+    public static $title = 'name';
+
+    /**
+     * The columns that should be searched.
+     *
+     * @var array
+     */
+    public static $search = [
+        'name',
+        'description',
+    ];
+
+    /**
+     * Get the fields displayed by the resource.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return array
+     */
+    public function fields(Request $request)
+    {
+        return [
+            new Panel('Basic Information', $this->basicFields()),
+
+            new Panel('Communications', $this->commFields()),
+
+            new Panel('Controls', $this->controlFields()),
+
+            new Panel('Metadata', $this->metaFields()),
+
+            HasMany::make('User', 'members'),
+        ];
+    }
+
+    protected function basicFields()
+    {
+        return [
+            Text::make('Name')
+                ->sortable()
+                ->rules('required', 'max:255'),
+
+            Textarea::make('Description')
+                ->hideFromIndex()
+                ->rules('required'),
+        ];
+    }
+
+    protected function commFields()
```

Comment:
Is there any way to add supporting text/description to the values of these fields to assist the user in properly filling them out?

Reply:
No hint text supported although we could rename the fields if something else would be clearer.

Most of them have reasonably self-explanatory names, and generally speaking, people who are not developers will not be filling them out.